packages/feedback-spa/src/mocks/options.js

```diff
@@ -0,0 +1,26 @@
+export const options = [
+  {
+    id: 0,
+    name: 'Mod Handovers'
+  },
+  {
+    id: 1,
+    name: 'Outages'
+  },
+  {
+    id: 2,
+    name: 'DSAL'
+  },
+  {
+    id: 3,
+    name: 'Feedback'
+  },
+  {
+    id: 4,
+    name: 'Projects & Initiatives'
+  },
+  {
+    id: 5,
+    name: 'Service Status'
+  }
```

Comment:
Please make sure that these options are valid from now onwards

Reply:
@riginoommen  file was js i converted that inot .ts   is that your mean ?

Reply:
i meant about the options like Service status, Projects & Inititives like that. Ensure that data that you push is valid

Reply:
I discussed same with Sayak as he informed right now it will be like this latter i will be dynamic data instead of hardcore, for now its fine.